DigitalLearningSolutions.Web/Views/LearningMenu/PostLearning/PostLearning.cshtml

```diff
@@ -43,8 +43,7 @@
       @Model.AssessmentStatus
       @if (Model.ScoreInformation != null)
       {
-        <span class="nhsuk-u-visually-hidden">, your score: </span>
-        @Model.ScoreInformation
+        @:(<span class="nhsuk-u-visually-hidden">Your score: </span>@Model.ScoreInformation)
```

Comment:
So this still has ":(" from what I can tell? I don't think the ":" should be there? 🙂

Reply:
That semi colon is part of the `@:` Razor syntax that converts the open bracket to plaintext as opposed to parsing it as C#.

Reply:
Ah I see! 🙂